modules/efk-logging-kibana-scaling.adoc

```diff
@@ -39,16 +39,7 @@ spec:
       type: "kibana"
       kibana:
         replicas: 1 <1>
-        resources:
-        proxy:
-          resources: <2>
-            limits:
-              cpu:
-              memory:
-            requests:
-              cpu:
-              memory:
 ---- 
 
 <1> Specify the number of Kibana nodes.
-<2> Specify the CPU and memory limits for each node.
+
```

Comment:
I don't expect the need to adjust the proxy but you could leave some note here, but it's not per node but for the proxy.

Reply:
@jcantrill I had removed this line from the file and created a new file for memory requests. 
https://github.com/openshift/openshift-docs/pull/14210/files#diff-54aee4af720c50efd3731f628998fb35R59